src/ExtensionsSample/App.config

```diff
@@ -1,9 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <connectionStrings>
```

Comment:
You're probably removing this because you don't want to forget not to put your stuff there? I have this here just because this sample is intended to be a regular WebJobs SDK project, which usually allows for these settings. I'd just leave these.